dashboard/src/components/LoadingWrapper/LoadingWrapper.tsx

```diff
@@ -1,7 +1,15 @@
 import * as React from "react";
+
 import LoadingSpinner from "../LoadingSpinner";
+import LoadingPlaceholder from "./LoadingPlaceholder";
+
+export enum LoaderType {
+  Spinner,
+  Placeholder,
+}
 
 export interface ILoadingWrapperProps {
+  type?: LoaderType;
```

Comment:
I am a little bit worried about using such an overloaded term `type`. It does not seem a reserved word in JS though.